torch/_dynamo/guards.py

```diff
@@ -1850,14 +1837,23 @@ def TENSOR_MATCH(self, guard: Guard, value=None):
                     else:
                         code.append(f"{tensor_name}.{term} == {real_value}")
             else:
-                self.tensor_check_examples.append(value)
-                self.tensor_check_names.append(tensor_name)
-                self.tensor_check_guards.append(guard)
```

Comment:
There's a refactor going on here that makes it hard to see the substantive change that happened here

Reply:
I can pull the refactor in a separate PR.